ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -893,6 +930,15 @@ def wait_for_ssh_threads():
                 # verify there are no interface flaps after warm boot
                 self.neigh_lag_status_check()
 
+        if self.reboot_type == 'fast-reboot' and not self.kvm_test:
+            self.no_cp_replies = self.extract_no_cpu_replies(self.upper_replies)
+            if self.no_cp_replies < 0.95 * self.nr_vl_pkts:
+                self.fails['dut'].add("Dataplane didn't route to all servers, when control-plane was down: %d vs %d" % (self.no_cp_replies, self.nr_vl_pkts))
+
+        if self.kvm_test and (self.no_control_stop - self.no_control_start) > datetime.timedelta(seconds=self.test_params['graceful_limit']):
```

Comment:
Do we need this check for control plane downtime here? The test already checks `wait_dut_to_warm_up`. The difference is that you expect the control plane to come back in `graceful_limit` - 240s,  and  the existing test checks warm up timeout for both control and data plane in `warm_up_timeout_secs` - 300s.

Additionally, there is a check above for:

`self.no_routing_stop - self.reboot_start > datetime.timedelta(seconds=self.test_params['graceful_limit']):`

Reply:
Removed the redundant check for the control plane.